### PR TITLE
Flags and Banners

### PR DESCRIPTION
**Pull request by LordFowl**
Adds flags and banners to the custom loadout section. Sprites based off of B12's SolGov flag, which has also been ported.

Loredevs are free to contact me if they have any flag designs they would like to see incorporated. I have only incorporated the major designs available immediately on the wiki. Namely, Dominia, Elyra, Izweski Hegemony, Jargon Federation, NanoTrasen, and SolGov

Image thereof;
image

### CODE REVIEW

**Comment by LordFowl** [comment]
Conflict is resolved

File: .gitignore [hunk 1]
```diff
@@ -1,18 +0,0 @@
-#ignore misc BYOND files
```
Comment (Arrow768) [inline .gitignore]: I dont think the deletion of the .gitignore and the .gitconfig is intentional ?

**Comment by LordFowl** [comment]
This is better.

**Comment by Arrow768** [comment]
Now you have a merge conflict again

**Comment by LordFowl** [comment]
A merge conflict is the least of my issues.

**Comment by Arrow768** [comment]
Right, just checked the modified files again. 
+317,745 −54,310 in 143 files

**Pull request closed without merging**